pkg/yurtctl/cmd/convert/convert.go

```diff
@@ -104,6 +106,9 @@ func NewConvertCmd() *cobra.Command {
 		"The yurt-tunnel-agent image.")
 	cmd.Flags().BoolP("deploy-yurttunnel", "t", false,
 		"if set, yurttunnel will be deployed.")
+	cmd.Flags().String("pod-manifest-path",
+		"/etc/kubernetes/manifests",
+		"Path to the directory containing static pod files.")
```

Comment:
how about add edge node info? like this:
Path to the directory on edge node containing static pod files.

Reply:
Adjusted